playbook-website/lib/pb_doc_helper.rb

```diff
@@ -27,33 +27,58 @@ def nav_hash_array(link)
 
     # Deal with lists of kits, used in Playbook doc and Externally
     # rubocop:disable Style/StringConcatenation
-    def pb_kits(type: "rails", limit_examples: false, dark_mode: false, method: get_kits)
```

Comment:
Isn't `method` used by the docs in Nitio? If so I suggest testing with an alpha.

Reply:
Yes. @nickamantia We'll want to test the 2nd & 4th search results[ here](https://github.com/search?q=repo%3Apowerhome%2Fnitro-web%20pb_kits&type=code) . Those (the dev docs page) depend on this method.

Reply:
@jasoncypret Clarification: we are only removing the get_kits method that is being called on the pb_kits method here. Nick is testing the alpha now, but it shouldn't be affected.